SolarSharp.Interpreter/Security/FileSystemSecurity.cs

```diff
@@ -144,7 +144,10 @@ public FilePermissions GetFilePermissions(string filePath)
                     }
 
                     // Also try to extract relative path from full path if it's absolute
-                    if (Path.IsPathFullyQualified(filePath))
+                    // TODO: Note this doesn't mean it's fully absolute, but this class likely needs an update
```

Comment:
These are not 100% replacements yes.  But this sort of folds into my other comments on preferring the filtering to be simpler and not as expensive as this.  Any sort of loop like this scares me for common ops.

Cache sort of just kicks the can down the road.

So I think this gets solved with my recommendation (from discord) of a cleaner wildcard, so I'm opting for this **because** IsPathFullyQualified does not exist in netstandard2.0.